packages/wp-now/src/wp-now.ts

```diff
@@ -38,7 +38,8 @@ async function applyToInstances(phpInstances: NodePHP[], callback: Function) {
 }
 
 export default async function startWPNow(
-	options: Partial<WPNowOptions> = {}
+	options: Partial<WPNowOptions> = {},
+	emscriptenOptions = {}
```

Comment:
This exposes a huge API that will move forward and change independently of Playground and without regard for Playground consumers. Let's expose a single option like `onStdout` instead.